cob_openni2_tracker/launch/body_tracker_nodelet.launch

```diff
@@ -21,7 +21,7 @@
 
 
 	<node pkg="nodelet" type="nodelet" name="body_tracker" args="load cob_openni2_tracker/body_tracker_nodelet /$(arg camera_name)/$(arg camera_name)_nodelet_manager" output="screen" cwd="ROS_HOME">
-		<param name="camera_frame_id" type="str" value="$(arg camera_name)_link" /> <!-- $(arg camera_name)_depth_optical_frame -->
+		<param name="camera_frame_id" type="str" value="$(arg camera_name)_rgb_frame" />   <!-- "$(arg camera_name)_link" -->
```

Comment:
Hi.

That is strange, I can't recall if I made the change, if is my change then probably either was some debugging I did to find out a way to get the tf frames or by mistake.

You can safely ignore this change.